packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -3077,6 +3093,10 @@ export function hydrateProperties(
     }
   }
 
+  if (props.onCommand != null) {
```

Comment:
The target of these events won't be the element with the `onCommand` listener but the element that is the designated `commandFor` element. Not sure how to model that with our current event infrastructure. That doesn't explain why it doesn't  work for `commandFor="app"` though.

Reply:
That makes sense right? You do `<div id="foo" onCommand={handle}><button commandFor="foo" command="--foobar">...</button>` and clicking the button causes the browser to fire a (non-bubbling) command event on the foo div. That shouldn't need special handling in react right?

Reply:
Though command events don't bubble, so I'm guessing I'm missing something somewhere to make the non capturing listener work with React's delegated event system?

Reply:
That's why you have to attach the listener to the dom element referenced in `commandFor` not the one where we want to listen for `command` events.